.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,13 +8,13 @@ about: Create a report to help us improve
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behaviour:
+Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behaviour**
+**Expected behavior**
```

Comment:
Can you move these unrelated changes out of this PR, please.

Reply:
Gone